portafly/src/App.tsx

```diff
@@ -0,0 +1,90 @@
+import 'react-app-polyfill/ie11'
+import '@patternfly/react-core/dist/styles/base.css'
```

Comment:
This file should be imported before any PF4/React component, in [portafly/src/index.tsx](https://github.com/3scale/porta/pull/1626/files#diff-fc8471bdbf51f64675a6cdb8f3fd55d0R3)